playbooks/roles/os_temps/tasks/main.yml

```diff
@@ -41,11 +41,17 @@
 - name: "Create project {{ openshift_project }}"
   import_tasks: get_set_project.yml
 
-# Import_tasks jq for querying container config files
+# Check if jq is already installed so we can skip installation
+- name: "Check if jq is already installed"
+  command: jq --version
```

Comment:
this works too but why don't you do an rpm query in command module? The reason I ask is this may fail if sudo is needed, etc..

Reply:
I went with that at first, but I wanted it to be a bit more os agnostic.

Reply:
nice, agree with this one re failure at this point :) But its basically user perms vs os, and we only support rpm as of now if I am not wrong, we could let it be then..